### PR TITLE
Add Waaila connector

### PR DESCRIPTION
**Pull request by matesmac**

### CODE REVIEW

**Review by dougelliott — DISMISSED** [review]

File: certified-connectors/Waaila/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,881 @@
...
+                  }
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/v1/depot/3db88c3b-fd01-414c-a2aa-90387a54b350/testsuite/5b3047c7-b05b-4049-91a2-6133b333c204": {},
```
Comment (sriyen-msft) [inline certified-connectors/Waaila/apiDefinition.swagger.json:342]: @matesmac Please remove this empty operation from the swagger as it won't add any functionality to the connector.

File: certified-connectors/Waaila/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
```
Comment (sriyen-msft) [inline certified-connectors/Waaila/apiProperties.json:3]: @matesmac  The connectionParameters property has mentioned with empty body in apiProiperties.json file. Empty connection object will be created automatically without any input from an user. It is less secure to use empty connection parameters. We usually suggest partners to have right connection. Please let us know the reason to keep no authentication for your connector.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Waaila/apiDefinition.swagger.json (hunk 1 above)
Comment (matesmac) [reply certified-connectors/Waaila/apiDefinition.swagger.json]: @sriyen-msft Removed

File: certified-connectors/Waaila/apiProperties.json (hunk 2 above)
Comment (matesmac) [reply certified-connectors/Waaila/apiProperties.json:3]: @sriyen-msft Thank you for the feedback. Actually the API is using authentication. It uses a Bearer token in the request header. 

As the token is very long and we needed to make the experience easier for our users, they generate a shorter time-limited API key in the Waaila web application. This is used in one of the operations (in combination with user email) to retrieve the full Bearer token that is required for all the other operations (the `Wauth` parameter in `GetDepots` and `GetTestsuite`). 

We originaly wanted to use the default Authentication header for this purpose but it is not accessible after the connection is made and therefore not suitable for our use case as it is not possible to fill it with a response value of one of the actions. This is why it seems like there is no authentication eventhough there is. 

We actually consulted the issue with our Microsoft Partner Technical Architect before submitting the connector and he suggested the workflow that is used. 

How we use the connector in our own Power App:

- Have a form in the app for filling out user email and API key (generated in Waaila web app with time-limited validity)
- Filled form triggers `GetToken` action. Retrieved token is saved in the memory.
- App uses other included actions (`GetDepots` and `GetTestsuite`) with `Wauth` parameter prefilled from the app memory.

**Review by sriyen-msft — APPROVED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**